api_app/views.py

```diff
@@ -60,7 +60,7 @@
     TagSerializer,
 )
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("__name__")
```

Comment:
why this change?

Reply:
I changed it while I was trying to add a console logger. I was having some problem with log file to see the recent changes. I will revert it back.